### ISSUE TITLE
Add Invite user by email modal

### ISSUE THREAD

**Issue by ninetteadhikari** [hikaya-io/dots-frontend#9]
![image](https://user-images.githubusercontent.com/13760198/78457355-db5bd280-76a9-11ea-8374-3a03a9a54d27.png)

**Comment by aryamanpuri** [comment]
Hi @ninetteadhikari, I would like to work on this issue.

**Comment by ninetteadhikari** [comment]
Sure go for it. If you have questions please reach out. We are using Element UI for design.

**Comment by aryamanpuri** [comment]
Hi, I was able to build the project.
Can you give a brief about what to be done on this issue?

**Comment by michaelbukachi** [comment]
Hi @aryamanpuri We can't tackle this issue yet because it's dependent on #8 .

**Comment by aryamanpuri** [comment]
Hi, @michaelbukachi Thanks for the clarification.
Is there any issue I can take up now ?

**Comment by michaelbukachi** [comment]
There's a couple of changes we need to make first. I'll finish them first then let you know.

**Comment by ninetteadhikari** [comment]
Agreed with Michael! @aryamanpuri I suggest you look at supporting to build UI components and not do this right now.
UI components: https://github.com/hikaya-io/vue-ui-components

**Comment by andrewtpham** [comment]
@michaelbukachi This screen is now ready for development and I believe the UI components are all ready:
* Input fields
* buttons
* Workspace setting card

**Comment by michaelbukachi** [comment]
Great! Let me publish the library.
